### ISSUE TITLE
Aarlo not working in 103

### ISSUE THREAD

**Issue by arsaboo** [twrecked/hass-aarlo#131]
Just updated to the beta and started seeing
```bash
2019-12-06 09:54:47 ERROR (MainThread) [homeassistant.components.alarm_control_panel] Error while setting up platform aarlo
Traceback (most recent call last):
  File "/srv/homeassistant/lib/python3.7/site-packages/homeassistant/helpers/entity_platform.py", line 150, in _async_setup_platform
    await asyncio.wait_for(asyncio.shield(task), SLOW_SETUP_MAX_WAIT)
  File "/usr/lib/python3.7/asyncio/tasks.py", line 442, in wait_for
    return fut.result()
  File "/home/homeassistant/.homeassistant/custom_components/aarlo/alarm_control_panel.py", line 100, in async_setup_platform
    base_stations.append(ArloBaseStation(base_station, config))
TypeError: Can't instantiate abstract class ArloBaseStation with abstract methods supported_features
```

**Comment by twrecked** [comment]
I have a fix. Just making sure it works across 102 and 103...

**Comment by arsaboo** [comment]
Let me know if you want me to test anything.

**Comment by twrecked** [comment]
I just checked in the change. Should be available in HACS soon.

https://github.com/twrecked/hass-aarlo/releases/tag/v0.6.3

**Comment by arsaboo** [comment]
Excellent...can confirm the issue is fixed in 0.6.3 👍
